test/Constraints/optional.swift

```diff
@@ -574,15 +574,14 @@ do {
   func passOptional(value: (any P)?) {
     takesP(value)
     // expected-error@-1 {{value of optional type '(any P)?' must be unwrapped to a value of type 'any P'}}
-    // expected-note@-2 {{coalesce using '??' to provide a default when the optional value contains 'nil'}}
-    // expected-note@-3 {{force-unwrap using '!' to abort execution if the optional value contains 'nil'}}
+    // expected-note@-2 {{coalesce using '??' to provide a default when the optional value contains 'nil'}} {{17-17= ?? <#default value#>}}
+    // expected-note@-3 {{force-unwrap using '!' to abort execution if the optional value contains 'nil'}} {{17-17=!}} 
   }
   func passLayersOfOptional(value: (any P)??) {
-    // FIXME(diagnostics): Consider recording multiple ForceUnwrap fixes based on number of optionals 
     takesP(value)
-    // expected-error@-1 {{value of optional type '(any P)??' must be unwrapped to a value of type '(any P)?}}
-    // expected-note@-2 {{coalesce using '??' to provide a default when the optional value contains 'nil'}}
-    // expected-note@-3 {{force-unwrap using '!' to abort execution if the optional value contains 'nil'}}
+    // expected-error@-1 {{value of optional type '(any P)??' must be unwrapped to a value of type 'any P'}}
+    // expected-note@-2 {{coalesce using '??' to provide a default when the optional value contains 'nil'}} {{17-17= ?? <#default value#>}}
```

Comment:
This is the problematic part - suggesting a single `??` is incorrect in this case, it has to be `(... ?? ...) ?? ...`...

Reply:
I have changed to not emit coalesce fix-it for more than one level. More than that looks like a too complicated chain of defaults to suggest. Let me know what you think :)